debian/tests/docker-in-lxd

```diff
@@ -165,22 +165,54 @@ lxc exec docker --env DEBIAN_FRONTEND=noninteractive -- apt-get full-upgrade -y
 lxc restart docker
 sleep 5
 
+if [ -n "${AUTOPKGTEST_TMP}" ]; then
+    # When running autopkgtest locally, the binary packages being
+    # tested are placed into the temporary autopkgtest directory, and
+    # then apt installs them from there.  In order to be able to
+    # install these packages, we must also copy them to the lxd
+    # container.  If we don't do this here, the docker.io package
+    # installed inside the container will come from the archive, which
+    # is not what we want.  This needs to be done after the "lxc
+    # restart" command because /tmp is emptied after the reboot.
+    autopkgtest_tmp_root=$(realpath "${AUTOPKGTEST_TMP}/../")
+
+    lxc exec docker -- rm -rf "${autopkgtest_tmp_root}"
+    lxc exec docker -- mkdir -p "${autopkgtest_tmp_root}"
+    tar -cC "${autopkgtest_tmp_root}" . | lxc exec docker -- tar -xC "${autopkgtest_tmp_root}"
+
+    lxc exec docker -- apt-get update
+fi
+
 lxc exec docker --env DEBIAN_FRONTEND=noninteractive -- apt-get install docker.io -y
 
 # Now basically run the simplest possible test inside the container.
 
 case ${arch} in
     amd64)
-        image=hello-world;;
+        image_hello_world=hello-world
+        image_ubuntu=ubuntu:devel;;
     armhf|i386|s390x)
-        image=${arch}/hello-world;;
+        image_hello_world=${arch}/hello-world
+        image_ubuntu="${arch}/ubuntu:devel";;
```

Comment:
This one is also not quite right -- see https://hub.docker.com/r/armhf/ubuntu vs https://hub.docker.com/r/arm32v7/ubuntu :disappointed:

Reply:
Thanks; fixed locally.